packages/admin/config/filament.php

```diff
@@ -207,10 +207,12 @@
         ],
         'sidebar' => [
             'is_collapsible_on_desktop' => false,
+            'hamburger_icon_on_desktop' => false,
```

Comment:
```suggestion
            'has_hamburger_icon_on_desktop' => false,
```

Reply:
What about `'use_hamburger_icon_on_desktop'` since it is a replacement instead of an addition to existing chevron.